sentry/src/main/java/io/sentry/SentryTransaction.java

```diff
@@ -238,7 +238,8 @@ Date getTimestamp() {
   @Override
   @Nullable
   public SpanStatus getStatus() {
-    return this.getContexts().getTrace().getStatus();
+    final SpanContext trace = this.getContexts().getTrace();
+    return trace != null ? trace.getStatus() : null;
```

Comment:
why not `context.getStatus()`?
I could set a status and check for it, why do I need to wait the transaction to be finished?

Reply:
Thanks Manoel, good point.